pxd_compat.h

```diff
@@ -22,7 +22,7 @@
 #define HAVE_BVEC_ITER
 #endif
 
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(6,11,0) || (LINUX_VERSION_CODE >= KERNEL_VERSION(5,14,0) && defined(__EL8__))
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6,11,0) || (LINUX_VERSION_CODE >= KERNEL_VERSION(5,14,0) && defined(__EL8__) && ! (defined(__RHEL_GT_94__)))
```

Comment:
isnt this defined in pxd.c? then how would this be effective outside pxd.c

you need a new macro 9.6 
see PR: https://github.com/portworx/px-fuse/pull/374